pages/about.html

```diff
@@ -0,0 +1,203 @@
+                <a class="nav-link active" aria-current="page" href="./pages/about.html">About</a>
+              </li>
+              <li class="nav-item me-lg-5">
+                <a class="nav-link" href="#">Program</a>
+              </li>
+              <li class="nav-item me-lg-5">
+                <a class="nav-link" href="#">Join</a>
+              </li>
+              <li class="nav-item me-lg-5">
+                <a class="nav-link" href="#">contact</a>
+              </li>
+              <li class="nav-item me-lg-5 p-1 border border-success">
+                <a class="nav-link text-success" href="#">
+                  Log Out
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+    </header>
+    <section class="container headline mt-4">
+      <nav aria-label="breadcrumb">
+        <ol class="breadcrumb justify-content-end me-70">
+          <li class="breadcrumb-item">
+            <a href="/" class="text-dark">
+              <i class="fas fa-home"></i>
+            </a>
+          </li>
+          <li class="breadcrumb-item active" aria-current="page">
+            About
+          </li>
+        </ol>
+      </nav>
+      <div class="content text-center mt-5">
+        <h1>
+          <span class="d-lg-none h2 d-block">"Hello Future Programmers"</span>
+          Web School
+        </h1>
+        <p class="m-auto">
+          We are a community of teachers and students. We work togther to create creative people who can
+          make awesome websites. So we have a special curriculum to make you professional web developer.
+          We will teach you the basics which are html, css, and js. After that you will learn framework for css
+          which is bootstrap 5. and a framework for js which is reactJS.
+        </p>
+        <div class="contact mt-5">
+          <h2 class="h6">
+            Please contact us if you have any further questions about web school!
+          </h2>
+          <a href="#" class="text-dark text-decoration-underline">
+            webscholl@gmail.com &raquo;
+          </a>
+        </div>
+      </div>
+      <div class="description mb-5 text-center">
+        <h2 class="h3">
+          What Is Web School?
+        </h2>
+        <p class="m-auto mb-5">
+          Are you tired of wasting your time and searching for useless courses and lost a lot of time? 
+          With Web School Academy, you will learn in the right way and with detailed academic explanations 
+          that explain everything in boring and smooth detail with a promise to understand everything 
+          no matter how difficult it is.
+          We exist 5 years ago. There are a lot of students who graduated from our school. And built their career.
+          And some of them have their own companies. Join now its time to make your success
+        </p>
+        <img src="../imgs/logo1.jpg" alt="logo" class="img-fluid"/>
+      </div>
```

Comment:
- [ ] This section must have a different background than the last section, and you missed the border around the logo, please, fix it.

Your design:

![imagen](https://user-images.githubusercontent.com/60631456/132545564-640b581a-d1e3-4c66-a507-cb92a98743b0.png)

Original design:

![imagen](https://user-images.githubusercontent.com/60631456/132545618-3ae6c757-a8a0-47e6-b878-018ca51276bf.png)